### ISSUE TITLE
Description for use of GitHub Maven packages is misleading

### ISSUE THREAD

**Issue by Marcono1234** [github/docs#1883]
### What article on docs.github.com is affected?
https://docs.github.com/en/free-pro-team@latest/packages/using-github-packages-with-your-projects-ecosystem/configuring-apache-maven-for-use-with-github-packages#installing-a-package

### What part(s) of the article would you like to see updated?
The section "Installing a package" is misleading. While other dependency management tools for other programming languages, such as npm for JavaScript, download executables which are executed as part of the build process and therefore "install" something, this is not the case for Maven (at least I have never seen it yet).
Instead `mvn install` builds the local project and then _installs_ the artifact which was created to the local repository. This artifact is mostly, if not completely, unrelated to the GitHub package which is used as dependency.

This might cause confusion for users new to Maven, therefore I would recommend:
- calling this section "Using a package" instead and updating the text
- omitting the code snippet which suggests to use `mvn install`

### Additional information
The same might also apply to the [respective page for Gradle](https://docs.github.com/en/free-pro-team@latest/packages/using-github-packages-with-your-projects-ecosystem/configuring-gradle-for-use-with-github-packages#installing-a-package), though I am not that familiar with Gradle.

This also applies to the GitHub packages page of a repository:
![Screenshot GitHub packages artifact](https://user-images.githubusercontent.com/11685886/101414747-fb056380-38e6-11eb-9408-8a7d8c8a77a1.png)
The second step "Install" is not needed.

Compare this with the following package hosts which do not have this step either and just provide the Maven / Gradle dependency snippet:
- https://bintray.com/bintray/jcenter/org.junit.jupiter%3Ajunit-jupiter
- https://mvnrepository.com/artifact/org.junit.jupiter/junit-jupiter-api/5.7.0
- https://search.maven.org/artifact/org.junit.jupiter/junit-jupiter/5.7.0/jar

**Comment by janiceilene** [comment]
Thanks for opening an issue @Marcono1234! I'll get this triaged for the right team to take a look 👀

**Comment by janiceilene** [comment]
Thanks @Marcono1234 we're going to leave the section header as it is for the sake of consistency!

**Comment by Marcono1234** [comment]
Thanks for the response. Do you mean consistency with packages for other programming languages?

The biggest issue I see with this is still that it will be misleading and irritating to users who are new to Maven.
Would an alternative at least be to change the command from `mvn install` to `mvn verify`? The `verify` phase happens immediately before `install` (see [Lifecycle Reference](https://maven.apache.org/guides/introduction/introduction-to-the-lifecycle.html#lifecycle-reference)), so the only step which would be missing is installing the built artifact (*not* the dependency) to the local repository. This step is most often not necessary anyways, and is certainly not necessary if the steps provided by GitHub only demonstrate how to download the dependency.
The advantage would be that the command would not be as misleading anymore as it is right now.

**Comment by janiceilene** [comment]
👋 @Marcono1234, I checked in with our engineers and you're right, the maven lifecycle has other stages that download a dependency, but the command we have listed (mvn install) is by far the most common way of downloading a maven dependency that is listed in a pom file. I don’t think our current instructions are confusing as they align with other instructions/tutorials I have seen on installing maven dependencies. Thanks so much for your passion and interest in GitHub docs!
